code/modules/antagonists/fugitive/hunter.dm

```diff
@@ -0,0 +1,166 @@
+//The hunters!!
+/datum/antagonist/fugitive_hunter
+	name = "Fugitive Hunter"
+	roundend_category = "Fugitive"
+	silent = TRUE //greet called by the event as well
+	var/datum/team/fugitive_hunters/hunter_team
+	var/backstory = "error"
+
+/datum/antagonist/fugitive_hunter/apply_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	update_fugitive_icons_added(M)
+
+/datum/antagonist/fugitive_hunter/remove_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	update_fugitive_icons_removed(M)
+
+/datum/antagonist/fugitive_hunter/greet(backstory)
+	switch(backstory)
+		if("space cop")
+			to_chat(owner, "<span class='boldannounce'>Justice has arrived. I am a member of the Spacepol!</span>")
+			to_chat(owner, "<B>The criminals should be on the station, we have special huds implanted to recognize them.</B>")
+			to_chat(owner, "<B>As we have lost pretty much all power over these damned lawless megacorporations, it's a mystery if their security will cooperate with us.</B>")
+		if("russian")
+			to_chat(src, "<span class='danger'>Ay blyat. I am a space-russian smuggler! We were mid-flight when our cargo was beamed off our ship!</span>")
+			to_chat(src, "<span class='danger'>We were hailed by a man in a green uniform, promising the safe return of our goods in exchange for a favor:</span>")
+			to_chat(src, "<span class='danger'>There is a local station housing fugitives that the man is after, he wants them returned; dead or alive.</span>")
+			to_chat(src, "<span class='danger'>We will not be able to make ends meet without our cargo, so we must do as he says and capture them.</span>")
+
+	to_chat(owner, "<span class='boldannounce'>You are not an antagonist in that you may kill whomever you please, but you can do anything to ensure the capture of the fugitives, even if that means going through the station.</span>")
+	owner.announce_objectives()
+
+/datum/antagonist/fugitive_hunter/create_team(datum/team/fugitive_hunters/new_team)
+	if(!new_team)
+		for(var/datum/antagonist/fugitive_hunter/H in GLOB.antagonists)
+			if(!H.owner)
+				continue
+			if(H.hunter_team)
+				hunter_team = H.hunter_team
+				return
+		hunter_team = new /datum/team/fugitive_hunters
```

Comment:
same here, why is the new team created AFTER you've assigned people to it?

Reply:
see my above comment on fugitive.dm